cross_list_courses/static/cross_list_courses/js/controllers/ListController.js

```diff
@@ -0,0 +1,107 @@
+(function(){
+    /**
+     * Angular controller for the cross listing page.
+     */
+    var app = angular.module('CrossListCourses');
+    app.controller('ListController',
+        ['$scope', '$http', '$timeout', '$document', '$window', '$compile','djangoUrl',
+        function($scope, $http, $timeout, $document, $window, $compile, $djangoUrl){
+
+            $scope.dtInstance = null;
+
+            $scope.formatCourse= function(course_instance) {
+                return course_instance.course.school_id.toUpperCase()+
+                                ' '+course_instance.course.registrar_code+
+                                '-'+course_instance.term.display_name+
+                                '-'+course_instance.course_instance_id;
+
+            };
+
+            $scope.dtOptions = {
+                ajax: function(data, callback, settings) {
+                    var url = $djangoUrl.reverse('icommons_rest_api_proxy',
+                                                   ['api/course/v2/xlist_maps/']);
+
+                    var queryParams = {
+                        offset: data.start,
+                        limit: data.length,
+                        include: 'course_instance',
```

Comment:
We can add `ordering` to this for the preferred default ordering (as discussed with @djbrooke offline):
```
ordering: '-last_modified_date';
```

Note that at least in QA we have some records with `last_modified_date` of `null`, which is entirely unhelpful, as these bubble to the top on this descending sort. In both QA and PROD pimary course instance 299994 is responsible for all of those null records. Might be worth asking @cmurtaugh / devops whether that data can be fixed, or whether we might expect nulls, in which case we need another more complex method for sorting.